sdk/communication/AzureCommunicationChat/Source/Generated/Models/AddChatParticipantsResult.swift

```diff
@@ -16,6 +16,7 @@ import Foundation
 // swiftlint:disable cyclomatic_complexity
 
 /// Result of the add chat participants operation.
+// TODO: Does not conform to equatable
```

Comment:
Was this TODO manually added by you @knvsl

Reply:
yes! was reminding myself so I can document the manual changes made - will open issue in autorest for them